signer/src/stacks/wallet.rs

```diff
@@ -495,4 +498,42 @@ mod tests {
 
         assert_eq!(wallet1.address(), wallet2.address())
     }
+
+    #[test]
+    fn public_key_order_independence_signer_wallet2() {
+        // Generally, for a stacks multi-sig wallet, the stacks address
+        // changes depending on the ordering of the given list of public
+        // keys. We don't want the address to depend on the ordering of
+        // keys. Check that the SignerWallet returns the same address
+        // regardless of the ordering of the keys given to it on
+        // construction.
+
+        let mut rng = rand::rngs::StdRng::seed_from_u64(51);
+        let mut public_keys: Vec<PublicKey> =
+            std::iter::repeat_with(|| Keypair::new_global(&mut OsRng))
+                .map(|kp| kp.public_key().into())
+                .take(50)
+                .collect();
+
+        let network_kind = NetworkKind::Testnet;
+        let pks1 = public_keys.clone();
+        let wallet = SignerWallet::new(&pks1, 5, network_kind, 0).unwrap();
+
+        let rotate_keys = RotateKeysTransaction {
+            txid: fake::Faker.fake_with_rng(&mut rng),
+            aggregate_key: wallet.aggregate_key(),
+            signer_set: public_keys,
+            signatures_required: wallet.signatures_required,
+        };
+
+        // Although it's unlikely, it's possible for the shuffle to not
```

Comment:
Was this something that you had intended to include or is it just here for mental documentation?

Reply:
ha, this entire file diff, from https://github.com/stacks-network/sbtc/pull/599/commits/f63ba4fa1b851757c9b37ae01f065325488fe49d) isn't supposed to be here. I was supposed to be in another branch. I'll remove it.